src/frequenz/sdk/timeseries/_serializable_ringbuffer.py

```diff
@@ -0,0 +1,91 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Ringbuffer implementation with serialization support."""
+
+# For use of the class type hint inside the class itself.
+from __future__ import annotations
+
+import pickle
+from datetime import datetime, timedelta
+from os.path import exists
+from typing import List, TypeVar
+
+import numpy as np
+import numpy.typing as npt
+
+from ._ringbuffer import OrderedRingBuffer
+
+Container = TypeVar("Container", List[float], npt.NDArray[np.float64])
```

Comment:
why not just:

``` suggestion
from ._ringbuffer import OrderedRingBuffer, Container
```

Reply:
I wasn't sure a specific `TypeVar` instance can or should be reused for more than one class

Reply:
For me it is also weird, and when a `TypeVar` is trivial, I prefer to define one per module (you can reuse it all you want though). But I guess when you restrict the types it can match like this it might make sense to re-use it, specially if both modules are tied together already.